tests/testthat/test-trace-collapse-children.txt

```diff
@@ -0,0 +1,17 @@
+Full:
+█
+└─f()
+  └─eval(quote(g()), env())
+    └─eval(quote(g()), env())
+      └─g()
+
+Collapsed:
+█
+└─f()
+  └─[ eval(...) ] ... +1
```

Comment:
I wonder if it would be better to do `eval()` here, to eliminate one set of `...`. It feels like it might be confusing currently

Reply:
But then it might be ambiguous with empty `f()` calls. If we use the more verbose "with 1 more call" there would only be one ellipsis.

Reply:
Yeah, sounds good.